alembic/versions/cd163a1b90b4_create_real_time_table.py

```diff
@@ -18,13 +18,12 @@
 def upgrade():
      op.create_table(
         'real_time',
-        sa.Column('key', sa.String(32), primary_key=True),
-        sa.Column('user_one_id', sa.String(64),  nullable=False),
-        sa.Column('user_two_id', sa.String(64), nullable=False),
-        sa.Column('latitude_one', sa.Float, nullable=False),
-        sa.Column('longitude_one', sa.Float, nullable=False),
-        sa.Column('latitude_two', sa.Float, nullable=False),
-        sa.Column('longitude_two', sa.Float, nullable=False),
+        sa.Column('waiting_user_id', sa.String(64),  nullable=False),
```

Comment:
yeah, because previous we add the primary key uuid. forgot to add primary key for this case.. will fix.